docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -204,6 +204,7 @@ This option is only available when you're editing the panel.
 | Show table header    | Show or hide column names imported from your data source. |
 | Frozen columns       | Freeze columns starting from the left side of the table. Enter a value to set how many columns are frozen. |
 | Cell height          | Set the height of the cell. Choose from **Small**, **Medium**, or **Large**. |
+| Max row height       | Define the maximum height for a row in the table. This may be useful when **Wrap text** is enabled for one or more columns. |
```

Comment:
```suggestion
| Max row height       | Define the maximum height for a row in the table. This can be useful when **Wrap text** is enabled for one or more columns. |
```